### ISSUE TITLE
Inconsistent CN links for replacement hotend heatsink fan

### ISSUE THREAD

**Issue by jaydenseric** [qidi-community/Plus4-Wiki#18]
There are 2 CN links for a replacement hotend heatsink fan that I've discovered so far, and both are different:

1. https://github.com/qidi-community/Plus4-Wiki/blob/9aa1d78b20cca1841f9ef52bfa6b111f09b86f95/README.md?plain=1#L90
2. https://github.com/qidi-community/Plus4-Wiki/blob/9aa1d78b20cca1841f9ef52bfa6b111f09b86f95/content/nozzle-clogging-tips/README.md?plain=1#L28

The second one doesn't select a particular variation of 4 product options, which is correct?

What is the best link to purchase? Let's make them consistent in all locations.

**Comment by stew675** [comment]
You are correct.  The AliExpress item does not match the other two, and in fact it appears to be changing over time.  I will remove that link.  I would suggest finding a GDSTime fan that matches the specs given on the EU shop page.  The US shop page is also correct, but it mistates the 12,000RPM as 1,200RPM, which is a typo.

**Comment by stew675** [comment]
@spooknik I am unable to edit your file https://github.com/qidi-community/Plus4-Wiki/blob/9aa1d78b20cca1841f9ef52bfa6b111f09b86f95/content/nozzle-clogging-tips/README.md

Could you make the correction?

**Comment by vega-d** [comment]
was fixed
